hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/HoodieSqlCommonUtils.scala

```diff
@@ -56,9 +56,10 @@ object HoodieSqlCommonUtils extends SparkAdapterSupport {
   })
 
   def getTableSqlSchema(metaClient: HoodieTableMetaClient,
```

Comment:
actually, getTableSqlSchema is only used in one place, and is set to true. So we can just use the current impl of getTableAvroSchema and add 
```
   if (timestamp == null) {
      return getTableAvroSchema(metaClient.getTableConfig().populateMetaFields());
    }
```
to the beginning of it

Reply:
and get rid of includeMetadataFields: Boolean = false, from getTableSqlSchema